core/bytes.go

```diff
@@ -25,3 +25,11 @@ func BytesToJSONString(bz []byte) (string, error) {
 
 	return string(formattedJSON), nil
 }
+func BytesToArray(bz []byte) ([32]byte, error) {
```

Comment:
@golangisfun123 

This needs a description comment to get the failed lint workflow to pass